bodhi/server/migrations/versions/2616c86d8ac6_add_indexes_for_status_and_request.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Till Maas
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+Add indexes for status and request
+
+Revision ID: 2616c86d8ac6
+Revises: 95ce24bed77a
+Create Date: 2017-10-26 20:45:23.152139
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+
+# revision identifiers, used by Alembic.
+revision = '2616c86d8ac6'
+down_revision = '95ce24bed77a'
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
```

Comment:
pydocstyle wants a docblock here. How about this:

```
"""
Add indices to the request and status columns on the update table, and add a compound uniqueness constraint on the Package's name and type.
"""
```

Similarly, pydocstyle wants a docblock on the ```downgrade()``` function below and we can probably just reverse the above sentence to say its removing those indices/constraints.